src/features/organization/mutations/deleteOrganizationRole.ts

```diff
@@ -0,0 +1,17 @@
+import { apiClient } from '@/config/apiClient';
+import { useMutation } from '@tanstack/react-query';
+
+export async function onDeleteOrganizationRole({ roleId }: { roleId: string }) {
+	const { data } = await apiClient.delete(`/Role/${roleId}` as '/Role/{id}', {
+		//@ts-expect-error would love to figure this out
+		id: roleId,
```

Comment:
Are you sure we need to send the id in the body in addition to sending it in the path?

Reply:
The api doc shows I need to